Classes/IndexQueue/PageIndexerRequest.php

```diff
@@ -209,7 +209,6 @@ public function getHeaders() {
 			'actions'   => implode(',', $this->actions),
 			'hash'      => md5(
 				$itemId . '|' .
-				$pageId . '|' .
```

Comment:
You mean this is a problem if you upgrade ... but if you have a clean index it should be fine.